src/modules/ManagedItems.js

```diff
@@ -30,9 +30,9 @@ export class ManagedItems {
         ? this.loadSetting('tableOptions', this.initialTableOptions)
         : this.tableOptions
     this._columns =
-      this.columns === undefined
-        ? this.loadSetting('columns', this.initialColumns)
-        : this.columns
+      this.tableColumns === undefined
```

Comment:
@ahochsteger Why did you changed this to `tableColumns`?

Reply:
It's just a naming change to make clear that it affects the table, since the `ManagedItems` may be used in different contexts.
It was meant to make it consistent with `tableOptions` and `tableHeaders`.
`filters` and `selected` are not specific to the table - they may also affect the grid layout and the mesh graph.